pkg/scuzz/sg_linux.go

```diff
@@ -112,24 +112,30 @@ type packet struct {
 	command commandDataBlock
 	status  statusBlock
 	block   dataBlock
+	word    wordBlock
 }
 
 // SGDisk implements a Disk using the Linux SG device
 type SGDisk struct {
 	f      *os.File
 	dev    uint8
 	packID uint32
-	master bool
 }
 
-// NewSGDisk returns a Disk that uses the Linux SCSI Generic Device
+// NewSGDisk returns a Disk that uses the Linux SCSI Generic Device.
+// It also does an Identify to verify that the target name is a true
+// lba48 device.
 func NewSGDisk(n string) (Disk, error) {
 	f, err := os.OpenFile(n, os.O_RDWR, 0)
 	if err != nil {
 		return nil, err
 	}
-	// TODO: figure out the dev and packID, or if we even need them.
-	return &SGDisk{f: f}, nil
+	d := &SGDisk{f: f}
+	if _, err := d.Identify(15000); err != nil {
```

Comment:
What's 15,000

Reply:
Oh.

Can you make timeout a property of SGDisk, since every command will have it?

Also, use time.Duration? Missed it last time.

Reply:
Yep, yeah, Duration!

I'll make it a property of sgdisk and probably use the option pattern for NewSGDIsk?. We'll see in the long term if people need to change it a lot for different operations in the same program but ... I doubt very much they will.  That's why I left it a function parameter but ... realistically .. I bet Nobody Cares :-)

Reply:
yeah, option pattern sgtm.